### PR TITLE
Update parameters.md

### PR DESCRIPTION
**Pull request by zhangjiekui**
tensorflow’s MyInit class (align to mxnet,pytorch)

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1553/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1553/

File: chapter_deep-learning-computation/parameters.md [hunk 1]
```diff
@@ -513,7 +513,10 @@ net[0].weight[:2]
 #@tab tensorflow
 class MyInit(tf.keras.initializers.Initializer):
     def __call__(self, shape, dtype=None):
-        return tf.random.uniform(shape, dtype=dtype)
+        data=tf.random.uniform(shape,-10, 10, dtype=dtype)
```
Comment (terrytangyuan) [inline chapter_deep-learning-computation/parameters.md:516]: Please place a space after comma. Same below and spaces between `>=`.

**Comment by mli** [comment]
Job d2l-en/PR-1553/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1553/

**Review by terrytangyuan — APPROVED** [review]
Thanks!

**Pull request merged**